[2023-01-05 03:39:53,354] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: btc_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-05T03:39:47.967706+00:00 [queued]>
[2023-01-05 03:39:53,368] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: btc_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-05T03:39:47.967706+00:00 [queued]>
[2023-01-05 03:39:53,368] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-05 03:39:53,368] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-05 03:39:53,368] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-05 03:39:53,386] {taskinstance.py:1377} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc_task> on 2023-01-05 03:39:47.967706+00:00
[2023-01-05 03:39:53,394] {standard_task_runner.py:52} INFO - Started process 2132 to run task
[2023-01-05 03:39:53,401] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'btc_data_pipeline_dag', 'create_cluster_dataproc_task', 'manual__2023-01-05T03:39:47.967706+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/btc_data_pipeline_dag.py', '--cfg-path', '/tmp/tmptrvi8x0t', '--error-file', '/tmp/tmp9lqivhpb']
[2023-01-05 03:39:53,402] {standard_task_runner.py:80} INFO - Job 54: Subtask create_cluster_dataproc_task
[2023-01-05 03:39:53,475] {task_command.py:370} INFO - Running <TaskInstance: btc_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-05T03:39:47.967706+00:00 [running]> on host 94450fd8b415
[2023-01-05 03:39:53,556] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=btc_data_pipeline_dag
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-05T03:39:47.967706+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-05T03:39:47.967706+00:00
[2023-01-05 03:39:53,558] {dataproc.py:590} INFO - Creating cluster: move-btc-data
[2023-01-05 03:39:53,560] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-01-05 03:42:42,830] {dataproc.py:534} INFO - Cluster created.
[2023-01-05 03:42:42,864] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=btc_data_pipeline_dag, task_id=create_cluster_dataproc_task, execution_date=20230105T033947, start_date=20230105T033953, end_date=20230105T034242
[2023-01-05 03:42:42,906] {local_task_job.py:156} INFO - Task exited with return code 0
